### PR TITLE
ShinezRTB Bid Adapter : initial release

### PR DESCRIPTION
**Pull request by saar120**
## Type of change

- [X] New bidder adapter  
## Description of change

Shinez Bid Adapter
maintainer email: [tech-team@shinez.io](mailto:tech-team@shinez.io)
```
{
        bidder: ‘shinezRtb’,
        params: {
          cId: ‘562524b21b1c1f08117fc7f9’,
          pId: ‘59ac17c192832d0011283fe3’,
          bidFloor: 0.0001,
          ext: {
            param1: ‘loremipsum’,
            param2: ‘dolorsitamet’
          }
     }
 }
```

## Other information

https://github.com/prebid/prebid.github.io/pull/4826

### CODE REVIEW

**Comment by assa-kariv** [comment]
@gwhigs @ChrisHuie can you please update what is the status here?

**Comment by assa-kariv** [comment]
@gwhigs we are waiting for you review for a couple of weeks now, could you please review?

**Review by osazos — COMMENTED** [review]
Hi @saar120 
Please could you check these points: 

- `digitrustid` eid exists in your code but not in your doc.
- the endpoint is not accessible: 
https://exchange.hb.sweetgum.io/prebid/multi/562524b21b1c1f08117fc7f9 returns `(failed)net::ERR_NAME_NOT_RESOLVED`
- writing a test for floor is a worth effort and should be added

File: test/spec/modules/shinezRtbBidAdapter_spec.js [hunk 1]
```diff
@@ -0,0 +1,634 @@
...
+      const serverResponse = utils.deepClone(SERVER_RESPONSE);
+      delete serverResponse.body.results[0].exp;
+      const responses = adapter.interpretResponse(serverResponse, REQUEST);
+      expect(responses).to.have.length(1);
+      expect(responses[0].ttl).to.equal(300);
+    });
+  });
+
+  describe('user id system', function () {
+    TEST_ID_SYSTEMS.forEach((idSystemProvider) => {
```
[the remark this answers is not in the document]
Comment (saar120) [reply test/spec/modules/shinezRtbBidAdapter_spec.js:508]: @osazos 
Added the test for digitrustid and changed the gppConsent to the correct one.
Replaced the endpoint to the correct one

[the remark this answers is not in the document]
Comment (osazos) [reply test/spec/modules/shinezRtbBidAdapter_spec.js:508]: @saar120, thx for your changes.

The endpoints still don't work on my side:

```shell
curl -I https://exchange.sweetgum.io
curl: (60) SSL certificate problem: unable to get local issuer certificate
```

```shell
curl -I https://sync.sweetgum.io/api/sync/image/\?cid\=\&gdpr\=0\&gdpr_consent\=\&us_privacy\=
curl: (6) Could not resolve host: sync.sweetgum.io
```

[the remark this answers is not in the document]
Comment (osazos) [reply test/spec/modules/shinezRtbBidAdapter_spec.js:508]: ping @saar120

**Review by osazos — APPROVED** [review]
LGTM

**Pull request merged**